### DISCUSSION TITLE
Serving Images from A Web Folder/Web Storage ?

### DISCUSSION THREAD

**Discussion by dukesx** [imgproxy/imgproxy#691] [Q&A]
Hi, 

Is it possible that i serve my images from a web folder or a web proxy, Currently i see that Amazon and GCS are supported but what if i have files on Bunny CDN storage ? Image-Kit , etc allow this function.

**Comment by DarthSim** [comment]
Hi!

As I see from the [API docs](https://docs.bunny.net/reference/manage-files), Bunny CDN gives access to the files by URL. As soon as it doesn't support private files, there's no need for a URL scheme for it.

**Comment by mstaack** [comment]
@DarthSim so imgproxy does not support Source By Url currently?

**Reply by DarthSim** [reply]
What gave you this idea? 🙂

**Reply by mstaack** [reply]
using bunnycdn, as storage source. only option currently could be by public URL. (but docs state its not supported)

otherwise custom storage implementation using https://github.com/jamesponddotco/bunnystorage-go or https://github.com/l0wl3vel/bunny-storage-go-sdk

**Reply by DarthSim** [reply]
> but docs state its not supported

Could you clarify which docs say that?

**Reply by mstaack** [reply]
could only find cloud storage or by local storage via path sources

**Reply by DarthSim** [reply]
That's because HTTP(S) doesn't need any special integration. Almost all the examples in the docs use HTTP(s) URLs: https://docs.imgproxy.net/usage/processing#source-url
